app/models/ability.rb

```diff
@@ -0,0 +1,49 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Handle the case where we don't have a current_user i.e. the user is a guest
+    # user ||= User.new
+    # can :read, :all, id: user.id
+
+    # if user.role == 'admin'
+    #   can :manage, :all
+    #   can :access, :rails_admin
+    # else
+    #   can :manage, Post, author_id: user.id
+    #   can :manage, Comment, author_id: user.id
+    #   can :read, User, id: user.id
+    # end
```

Comment:
- [ ] [OPTIONAL] Kindly, remove the commented code from your code base to make your code base clean.

Reply:
Yeah sure!..
:+1:  :rocket: